### ISSUE TITLE
nth($list, $n) overrides the $list argument 

### ISSUE THREAD

**Issue by pxl-dgn** [sass/libsass#2226]
The function `nth($list, $n)` overrides the actual $list argument: it replaces the css value (eg. `16px/18px`) with the calculated value (`0.88889`). Ruby sass and older versions of libsass do not exhibit this behavior.

nth.scss
```scss
$font: normal 16px/18px serif;
html { font: $font; }
$metrics: nth($font, 2);
body { font: $font; }
```

### Expected result:
```css
>sass nth.scss
html {
  font: normal 16px/18px serif; }

body {
  font: normal 16px/18px serif; }
```

version info:
```cmd
>sass --version
Sass 3.4.22 (Selective Steve)
```

### Actual result:
```css
>node-sass nth.scss
html {
  font: normal 16px/18px serif; }

body {
  font: normal 0.88889 serif; }
```

version info:
```cmd
>node-sass --version
node-sass       3.11.2  (Wrapper)       [JavaScript]
libsass         3.3.6   (Sass Compiler) [C/C++]
```

**Comment by pxl-dgn** [comment]
It seems the last correct version is 
```
node-sass       3.4.2   (Wrapper)       [JavaScript]
libsass         3.3.2   (Sass Compiler) [C/C++]
```

The next available (node-sass) version does have this issue.
```
node-sass       3.5.1   (Wrapper)       [JavaScript]
libsass         3.3.5   (Sass Compiler) [C/C++]
```

**Comment by xzyfer** [comment]
@pxl-dgn thanks for the report. I can confirm this is fixed in 3.4.0-RC1.
